templates

```diff
@@ -471,6 +471,8 @@
       name => "live_default",
       settings => [
         { key => "LIVE", value => "1" },
+        # The raw disk image or ISO is provided as HDD_2
+        { key => "NUMDISKS", value => "2" },
```

Comment:
If you're booting from a raw disk image, you'll get a different FBE, and in particular you won't be given the choice to Try or Reformat. Are you sure you don't want something the other way around: skip live/install tests when testing the raw disk image, and jump straight to completing the "real" FBE?

Reply:
This is not really anything to do with the difference between ISOs and raw disk images; the comment I put there is slightly incorrect (it should just say ‘ISO’).

The tests already look at the `LIVE` key and adjust their expectations of whether to do a live boot + reformat and install, or skip straight to FBE from the raw disk image, accordingly.

Reply:
(I think that’s a rambly way of saying I don’ŧ think your comment is relevant and I’m a bit confused about it. Sorry if I’ve misunderstood.)

Reply:
And here, again, I still don't understand. Why does the live-boot test need to be able to see the `.img.xz`?